### DISCUSSION TITLE
Missing Icons

### DISCUSSION THREAD

**Discussion by kaisei15** [darkreader/darkreader#9596] [Q&A] [answered]
EDIT: I submitted to Issues.

I get a lot of websites where the icons are missing. Attached is an example from https://www.harrypottermagiccasterwand.com/ when you click on watch trailer. This happens on many other websites too, but this is just the latest one that occurred and gave me motivation to come here to report. It is not always a video, just random buttons will not show up.

Anyone else experience this / know a fix?

Thanks
![2022-08-17 16_07_19-NVIDIA GeForce Overlay DT](https://user-images.githubusercontent.com/40921612/185259291-c2f63ea1-fb07-4e12-bf7e-a8f0d5010012.png)

**Comment by Gusted** [comment, marked as answer]
Do you have a custom font enabled?
